docs/content/en/docs/fleet-manager/distributedstorage-plugin.md

```diff
@@ -0,0 +1,366 @@
+rook-ceph-mgr-a-8456d8cc98-n54n2                                    3/3     Running            0          32h
+rook-ceph-mgr-b-67c9cc8c95-gk82m                                    3/3     Running            0          32h
+rook-ceph-mon-a-68659ffbfb-bqn6q                                    2/2     Running            0          32h
+rook-ceph-mon-b-85479654f8-n26vs                                    2/2     Running            0          32h
+rook-ceph-mon-c-6779986564-p8dmc                                    2/2     Running            0          32h
+rook-ceph-operator-b89ccd545-vlg2r                                  1/1     Running            0          32h
+rook-ceph-osd-0-996475cb6f-cfbwh                                    1/1     Running            0          32h
+rook-ceph-osd-1-67f5ff649c-vs5qt                                    1/1     Running            0          32h
+rook-ceph-osd-2-7d4c78b74b-l8jg5                                    1/1     Running            0          32h
+rook-ceph-osd-prepare-testpool-dev-linux-0001-jz766                 0/1     Completed          0          32h
+rook-ceph-osd-prepare-testpool-dev-linux-0003-t47st                 0/1     Completed          0          32h
+rook-ceph-osd-prepare-testpool-dev-linux-0004-kw94n                 0/1     Completed          0          32h
+rook-ceph-rgw-ceph-objectstore-a-5c4df48bbb-bf6jn                   2/2     Running            0          32h
+```
+
+## Persistent Volume Use Guide
+
+After rook opeartor and rook ceph cluster are installed, this chapter provides examples of using Block Storage, Filesystem Storage and Object Storage.
+
+### Block Storage Class Configuration
+
+Block storage allows a single pod to mount storage. This section how to create a block storage persistent volume kubernetes pod that uses the block storage provided by Rook.
+
+The `StorageClass` and `CephBlockPool` need to be created before the Rook can configure storage.This will allow Kubernetes to interact with the Rook when configuring persistent volumes.
+
+Run the following commands, can create a `CephBlockPool`:
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph
+spec:
+  failureDomain: host
+  replicated:
+    size: 3
+EOF
+```
+
+Run the following commands, can create a block mod `StorageClass`:
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: rook-ceph-block
+provisioner: rook-ceph.rbd.csi.ceph.com
+parameters:
+    clusterID: rook-ceph
+    pool: replicapool
+reclaimPolicy: Delete
+allowVolumeExpansion: true
+```
+
+There are a few things to note in the above block storage class configuration:
+
+- provisioner is configured in the format (operator-namespace).rbd.csi.ceph.com. Change "rook-ceph" provisioner prefix to match the operator namespace if needed.
+- `parametes.clusterID` is the namespace where the rook cluster is running.
```

Comment:
clusterID is namespace?

Reply:
In the documentation for rook, it is mentioned that the clusterID is the namespace where the rook cluster is running.
You can refer to the following link [rook documtation](https://rook.io/docs/rook/v1.11/Storage-Configuration/Block-Storage-RBD/block-storage/#provision-storage)